style.css

```diff
@@ -0,0 +1,308 @@
+
+body {
+  margin: 0;
+  padding: 24px;
+  font-family: 'Poppins', sans-serif;
+  display: flex;
+  flex-direction: column;
+  background-color: #f7f7f9;
+  box-sizing: border-box;
+  max-width: 375px;
+  overflow-x: hidden;
+}
+
+.bio-section {
+  margin-top: 5%;
+  display: flex;
+  flex-direction: column;
+  background-image: url(img/backg.png);
+  background-size: cover;
+  background-position: center;
+  margin-bottom: 250px;
+  margin-left: 0;
+  margin-right: 0;
+  width: 100%;
+  padding: 0 0 0 0;
+}
+
+.bio {
+  border: 2px transparent #fff;
+  border-bottom-left-radius: 11%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+  margin-bottom: 1px;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  cursor: pointer;
+  margin-left: 1%;
+  text-decoration: none;
+}
+
+.bio-header {
+  font-size: 49px;
+  font-weight: 800;
+  line-height: 60px;
+  align-items: center;
+  letter-spacing: 0.37px;
+  bottom: 20%;
+  color: #091e42;
+}
+
+.bio-text {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  max-width: 80vw;
+  top: 30%;
+  color: #344563;
+}
+
+.connect {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.connect-icons {
+  width: 200px;
+  display: flex;
+  justify-content: space-between;
+  align-content: flex-start;
+  padding-left: 1px;
+  padding-top: 1px;
+  margin: 1px;
+  list-style: none;
+}
+
+.connect-icons i {
+  margin: 0;
+  font-size: 21px;
+  color: #505f79;
+  height: 19.96px;
+  width: 19.99px;
+}
+
+.connect-icons i:hover {
+  font-size: 19px;
+  transition: 1s ease-in-out;
+  cursor: pointer;
+}
+
+/* Works Styles */
+.Works-Section {
+  max-width: 100%;
+  display: grid;
+  text-align: center;
+  padding: 0 0 0 0;
+  gap: 88px;
+  margin-bottom: 90px;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.project-1 {
+  border-radius: 16px;
+  background-color: white;
+  border: 1px solid lightgray;
+  padding: 15px;
+}
+
+.connect-2 {
+  color: #7f8cff;
+  font-size: 16px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+}
+
+.project-name {
+  text-align: left;
+  font-size: 20px;
+}
+
+.project-name h2 {
+  margin-top: 5px;
+  margin-bottom: 5px;
+  padding-right: 21px;
+  line-height: 40px;
+  color: #172b4d;
+}
+
+.project-languages {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 0 0;
+  padding: 0 0 0 0;
+  width: auto;
+}
+
+.project-links li:first-child a {
+  font-weight: 600;
+  text-transform: uppercase;
+  color: #2e4662;
+}
+
+.project-links {
+  display: flex;
+  justify-content: space-between;
+  max-width: 100vw;
+  margin: 0 0 0 0;
+  padding: 0 0 0 0;
+}
```

Comment:
Figma
![image](https://user-images.githubusercontent.com/78251927/229296455-0a8a165a-125f-40f4-b2ce-deba832feeeb.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/229296507-be28b732-d2aa-48b1-a9ef-3dda854dedcd.png)

Please kindly justify-content to the left to match the template on Figma 
```css
justify-content: left;
```